examples/source_separation/utils/dataset/librimix.py

```diff
@@ -0,0 +1,73 @@
+from pathlib import Path
+from typing import Union, Tuple, List
+
+import torch
+from torch.utils.data import Dataset
+
+import torchaudio
+
+SampleType = Tuple[int, torch.Tensor, List[torch.Tensor]]
+
+
+class LibriMix(Dataset):
```

Comment:
This dataset is also used in some superb tasks and I think would be worth supporting in `torchaudio/datasets` instead of the `examples` folder, where we can create a wrapper class in the examples directory if necessary for running source separation. Thoughts on this? This could be a follow up that doesn't need to be addressed in this PR

Reply:
Yeah sounds great. Let's make a list of the datasets we want to add for s3prl and have a team discussion?